app/controllers/idv/gpo_controller.rb

```diff
@@ -11,6 +11,19 @@ class GpoController < ApplicationController
     def index
       @presenter = GpoPresenter.new(current_user, url_options)
       @step_indicator_current_step = step_indicator_current_step
+      # Funnel::DocAuth::RegisterStep.new(current_user.id, current_sp&.issuer).
+      #   call(:usps_address, :view, true)
```

Comment:
I am open to dropping the commented out lines, but figured it was easier to leave them in. Could also have gone with a feature flags approach, but that seemed like a lot more temporary code and more complex testing